CONTRIBUTING.md

```diff
@@ -97,33 +97,36 @@ system internals, try asking in [`#rust-internals`][pound-rust-internals].
 
 Before you can start building the compiler you need to configure the build for
 your system. In most cases, that will just mean using the defaults provided
-for Rust. Configuring involves invoking the `configure` script in the project
-root.
+for Rust.
 
-```
-./configure
-```
+To change configuration, you must copy the file `src/bootstrap/config.toml.example`
+to `config.toml` in the project root, and change the settings provided.
```

Comment:
Do you mean ./build/config.toml when you say this? Or "directory from which you will be running the build", as the config.toml.example file states?